package.json

```diff
@@ -88,7 +88,7 @@
     "typescript": "^5.0.2",
     "unbuild": "^1.2.1",
     "vite": "workspace:*",
-    "vitepress": "^1.0.0-alpha.73",
+    "vitepress": "1.0.0-beta.1",
```

Comment:
For some reason when I use `^1.0.0-beta.1`, pnpm resolves to `1.0.0-draft.1` which doesn't work.